bin/README.md

```diff
@@ -11,14 +11,20 @@ determine their own directory based on the implementation langugage hooks availa
 For example from a bash-type shell:<br>
```

Comment:
This PR also deletes `RV32Zifencei_Extension_Instruction copy.xlsx`.
Can you confirm that this is intentional?

That ISA fencei vplan has _more_ info in it than the non-"copy" ISA fencei vplan.
But I agree that this PR makes the right move, in keeping the simpler ISA fencei vplan and deferring the fencei semantics details to core-specific vplans.

Reply:
Yes, this was intentional.  I assumed that including it was a fat-finger-issue.  Let me know if you wish to keep it.

Reply:
I too assumed it was a fat-finger-issue. I tip in favor of purging it like this PR does.

However, even if "ISA-general" vplans are more superficial than "core-specific" vplans (which might include micro architecture and implementation-specific decicions), the Vol I spec _does_ say that _"A FENCE.I instruction ensures that
a subsequent instruction fetch on a RISC-V hart will see any previous data stores already visible to the same RISC-V hart"_.
So it _could_ be that the non-"copy" vplan is a bit too thin.
But that is a methodology/philosophy question that is a bit on the side of the Python fixing of this PR.